webview/util.py

```diff
@@ -131,6 +136,10 @@ def _call():
 
         window.evaluate_js(code)
 
+    if window.frameless and not window.easy_drag and func_name == 'moveWindow':
```

Comment:
Imo drag region can be enabled by default as long as the region class is found. Frameless window should not be a prerequisite to it.
Otherwise looks good. Great job!

Reply:
Agreed, have enabled it by default in https://github.com/r0x0r/pywebview/pull/520/commits/1d7a5159cc6460aaaa8da5f5ea80ef31c206b104.